### DISCUSSION TITLE
Android shortcuts

### DISCUSSION THREAD

**Discussion by NePheus** [ionic-team/capacitor#3699] [Ideas]
Is there a way to use Android shortcuts with Capacitor?
https://developer.android.com/guide/topics/ui/shortcuts

There is an existing cordova plugin, maybe its possible to write a @ionic/native wrapper?
https://github.com/avargaskun/cordova-plugin-shortcuts-android#readme

**Comment by MNorgren** [comment]
This would be a great feature to have. Any updates or ideas?

**Comment by NePheus** [comment]
I created a capacitor 3 plugin for android shortcuts:
https://www.npmjs.com/package/capacitor-android-shortcuts
